Test_Sender_BMS/Test_Sender_BMS.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Sender_BMS;
+using Xunit;
+
+namespace Test_Sender_BMS
+{
+    public class Test_Sender_BMS
```

Comment:
Test of output format missing. The tests will check if one random number is between the limits. Even if you take wrong limits, it can pass due to randomness.
Check your code, not the built-in random-number-generator. We can safely assume that the built-in functions are working.